internal/config/main.go

```diff
@@ -59,6 +61,7 @@ var (
 		TLSInsecure:   flag.Bool("insecure", false, "Allow insecure certificates"),
 		TlsExtraCAs:   flag.String("extra-ca-certs", "", "comma-separated files containing extra PEMs to trust for TLS connections in addition to the system trust store"),
 		MaxBGPPaths:   flag.Int("max-bgp-paths", 10000, "How many BGP Paths to receive when counting routes, needs to be higher then the number of routes or metrics will not be generated"),
+		MaxVPNUsers:   flag.Int("max-vpn-users", 0, "How many VPN Users to receive when counting users, needs to be greater than or equal the number of users or metrics will not be generated (0 eq. none by default)"),
```

Comment:
What is the rational behind a complete abort if max-vpn-users is smaller than the results?     
 As user I would expect that we simply stop adding new metrics.

Reply:
The problem is that if we do that, we assume that the order of all entries are the same for every API request from the fortigate, or we would end up getting partial results that differ for every scrape.

This may hold today (no idea if it does), but may change anytime in the future.